metrics/internalrequest_test.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/testutil"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+var _ = Describe("Metrics InternalRequest", Ordered, func() {
+	BeforeAll(func() {
+
+		// We need to unregister in advance otherwise it breaks with 'AlreadyRegisteredError'
+		metrics.Registry.Unregister(InternalRequestAttemptConcurrentTotal)
+		metrics.Registry.Unregister(InternalRequestAttemptDurationSeconds)
+	})
+
+	var (
+		AttemptDurationSecondsHeader = inputHeader{
```

Comment:
I know they are like this in the release-service, but why are they capitalized? They are only used in this package.

Reply:
I have no idea why. Maybe there was a reason back then. I will correct it here, at least.